src/types.rs

```diff
@@ -196,6 +196,45 @@ pub struct GetMempoolInfo {
     pub unbroadcastcount: usize,
 }
 
+/// Response from `getrawmempool` with `verbose=true`.
+///
+/// The top-level map key is the txid, and the value contains detailed mempool info per tx.
+pub type GetRawMempoolVerbose = BTreeMap<Txid, MempoolEntry>;
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct MempoolEntry {
+    pub vsize: usize,
+    pub weight: usize,
+    #[serde(default)]
+    pub fee: Option<f64>, // DEPRECATED
+    #[serde(default)]
+    pub modifiedfee: Option<f64>, // DEPRECATED
+    pub time: u64,
+    pub height: usize,
+    pub descendantcount: usize,
+    pub descendantsize: usize,
+    #[serde(default)]
+    pub ancestorcount: usize,
+    pub ancestorsize: usize,
+    #[serde(default)]
+    pub ancestorfees: Option<f64>, // DEPRECATED
+    pub wtxid: String,
+    pub fees: Option<MempoolFeeBreakdown>,
+    pub depends: Vec<String>,
+    pub spentby: Vec<String>,
```

Comment:
I think we can do with
```suggestion
    pub depends: Vec<Txid>,
    pub spentby: Vec<Txid>,
```
Or maybe we need to tweak the `deserialize_txid` to have a version `deserialize_vector_txid`